vulcan/db/migrations/011_add_workspace_git_request.rb

```diff
@@ -0,0 +1,8 @@
+Sequel.migration do
+  change do
+    alter_table(:workspaces) do
+      add_column :git_request_by, String, null: false, default: "tag_or_sha"
```

Comment:
Is this column really necessary? Isn't it possible to distinguish a branch from a tag or sha just by format and/or via git? For example I think git checkout <branch, tag or sha> will work

Reply:
Yes, it is possible to do that, but it's saved for serving suggestions in the front-end's workspace creation UI.

It doesn't seem worth it to me to be recalculating this whenever someone hits a project dashboard?  This is primarily served by our shallower list_workspaces API that lists ALL workspaces.

Reply:
this is gone now!